index.js

```diff
@@ -62,6 +84,14 @@ function $asNull () {
   return 'null'
 }
 
+function $asInteger (i) {
+  if (isLong && isLong(i)) {
+    return i.toString()
+  } else {
+    return $asNumber(i)
+  }
+}
```

Comment:
This is already the case.

See https://github.com/fastify/fast-json-stringify/pull/30/files#diff-168726dbe96b3ce427e7fedce31bb0bcR32